File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/ycanas.py
def fizz_buzz(n):
    for i in range(1, n+1):
        by_three = i%3 == 0
        by_five  = i%5 == 0

        if by_three and by_five:
            print("fizzbuzz")
        
        elif by_three:
            print("fizz")

        elif by_five:
            print("buzz")

        else:
            print(i)

fizz_buzz(100)

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/ycanas.java
public class ycanas{
    public static void main(String[] args){
        for(int number = 1; number <= 100; number++){
            if(number % 3 == 0 && number % 5 == 0){
                System.out.println("fizzbuzz");
            }
            
            else if(number % 3 == 0){
                System.out.println("fizz");
            }
            
            else if(number % 5 == 0){
                System.out.println("buzz");
            }
            
            else{
                System.out.println(number);
            }
        }
    }
}
